x-pack/legacy/plugins/siem/public/pages/network/navigation/network_routes.tsx

```diff
@@ -31,17 +32,17 @@ export const NetworkRoutes = ({
   setQuery,
   setAbsoluteRangeDatePicker,
 }: NetworkRoutesProps) => {
-  const narrowDateRange = useCallback(
```

Comment:
why have you decided to remove `useCallback`?

Reply:
Nice catch, I was playing around with it and failed to add it back.